pkg/monitor/sqsevent/scheduled-change-event.go

```diff
@@ -76,13 +76,13 @@ func (m SQSMonitor) scheduledEventToInterruptionEvents(event *EventBridgeEvent,
 	}
 
 	if scheduledChangeEventDetail.Service != "EC2" {
-		err := fmt.Errorf("events from Amazon EventBridge for service (%s) are not supported", scheduledChangeEventDetail.Service)
-		return append(interruptionEventWrappers, InterruptionEventWrapper{nil, err})
```

Comment:
This error indicates something that requires action from the user, to update their configuration, rather than just providing a heads-up of an application oddity. With that perspective, I think keeping it as a (skippable) error makes more sense (I know I mentioned this idea in the first place! Sorry!)

It looks like we've recently added support for a skippable type of error: https://github.com/aws/aws-node-termination-handler/blob/37e989956e163b027cd2dcc04a58ce89a75244c6/pkg/monitor/sqsevent/sqs-monitor.go#L58 Switching to use this should be a pretty similar change 🙂